plugins/module_utils/foreman_helper.py

```diff
@@ -384,7 +403,8 @@ def _delete_entity(self, resource, current_entity, params):
         payload = {'id': current_entity['id']}
         if params:
             payload.update(params)
-        return self._resource_action(resource, 'destroy', payload)
+        self._resource_action(resource, 'destroy', payload)
+        return True, None
```

Comment:
isn't this identical to the previous code?

Reply:
No, the previous code returned the deleted entity. This now states: There is no entity (after deletion). We do not use the return value atm iirc.